eessi/testsuite/common_config.py

```diff
@@ -0,0 +1,58 @@
+perflog_format = '|'.join([
+    '%(check_job_completion_time)s',
+    '%(osuser)s',
+    '%(version)s',
+    '%(check_unique_name)s',
+    '%(check_info)s',
+    '%(check_system)s',
+    '%(check_partition)s',
+    '%(check_environ)s',
+    '%(check_exclusive_access)s',
+    '%(check_num_tasks)s',
+    '%(check_num_cpus_per_task)s',
+    '%(check_num_tasks_per_node)s',
+    '%(check_num_gpus_per_node)s',
+    '%(check_use_multithreading)s',
+    '%(check_modules)s',
+    '%(check_jobid)s',
+    '%(check_perfvalues)s',
+])
+
+format_perfvars = '|'.join([
+    '%(check_perf_var)s',
+    '%(check_perf_value)s',
+    '%(check_perf_lower_thres)s',
+    '%(check_perf_upper_thres)s',
+    '%(check_perf_unit)s',
+    ''  # final delimiter required
+])
+
+common_logging_config = [{
+    'level': 'debug',
+    'handlers': [
+        {
+            'type': 'stream',
+            'name': 'stdout',
+            'level': 'info',
+            'format': '%(message)s',
+        },
+        {
+            'type': 'file',
+            'name': 'reframe.log',
```

Comment:
As discussed, this log file does _not_ end up in the `prefix`  set for the system. While ` --save-log-files` copies this to the root of the `output` directory (which by default will be in `<prefix>/output`), it's not very nice as it _also_ leaves a log file in the current working directory.
A much cleaner structure would be:
```
<prefix>/output
<prefix>/staging
<prefix>/perflogs
<prefix>/logs
``` 
The first three are created this way. If we can convince the `file` log to end up in `<prefix>/logs`, we are done.

We discussed options on _how_ to do this. The idea was that `common_logging_config` should probably be a function that you can pass the `prefix` as an argument. Then, we can set 
```
'name': '<prefix_arg>/logs/reframe.log'
```
The function then returns the `common_logging_config`.

Reply:
the downside of this approach is that if the user specifies the prefix via `--prefix`, that will be ignored. we can check for `RFM_PREFIX` though.
should we open an issue about it?

Reply:
this is now implemented.
you can override the prefix specified in the config file with `RFM_PREFIX`, but as noted above, `--prefix` does not work for file logs.